python/ray/dag/input_node.py

```diff
@@ -57,7 +58,7 @@ class InputNode(DAGNode):
     ...     # dag_input["m1"], dag_input["m2"]
     """
 
-    def __init__(self, *args, _other_args_to_resolve=None, **kwargs):
+    def __init__(self, *args, return_types=None, _other_args_to_resolve=None, **kwargs):
```

Comment:
Could you (1) add a type hint to `return_types` and (2) describe it in the docstring?